pxf/Makefile

```diff
@@ -19,13 +19,13 @@
 default: all
 
 ifneq "$(HD)" ""
-    BUILD_PARAMS= -Dhd=$(HD)
+    BUILD_PARAMS+= -Dhd=$(HD)
 else
     ifneq "$(PXF_HOME)" ""
-        BUILD_PARAMS= -DdeployPath="$(PXF_HOME)"
+        BUILD_PARAMS+= -DdeployPath="$(PXF_HOME)"
     else ifneq "$(GPHOME)" ""
         PXF_HOME= "$(GPHOME)/pxf"
-        BUILD_PARAMS= -DdeployPath="$(PXF_HOME)"
+        BUILD_PARAMS+= -DdeployPath="$(PXF_HOME)"
```

Comment:
minor ;) -- you can move the BUILD_PARAMS line outside of if/else since it's the same line in both places.